rcl/test/rcl/test_timer.cpp

```diff
@@ -995,7 +995,7 @@ TEST_F(TestPreInitTimer, test_timer_info) {
   rcl_timer_call_info_t call_info;
   call_info.actual_call_time = 0;
   call_info.expected_call_time = 0;
-  int64_t period = RCL_MS_TO_NS(10);
+  int64_t period = RCL_MS_TO_NS(100);
```

Comment:
on 1st `rcl_timer_call_with_info`, it already misses the cycle, that eventually leads to the mismatch in line 1021.

Reply:
We could use mimick to mock rcl_clock_get_now.